src/frontend/shortlist/src/components/layout/profile/UserRelations.vue

```diff
@@ -8,7 +10,19 @@ export default {
   },
   methods: {
     sendInvite() {
-      this.$emit("addStudent", this.newEmail);
+      axios
+        //address needs change to coop
+        .post("https://api.shortlist.nyc/shortlists/send-invite", {
```

Comment:
```suggestion
        .post("https://api.shortlist.nyc/auth/send-invite", {
```
See here https://manage.shortlist.nyc/

Reply:
Roger that, change has been made